src/tests/ftest/datamover/posix_types.py

```diff
@@ -66,162 +66,129 @@
-
-        copy_list.append([
-            "UNS -> POSIX",
-            ["DAOS_UNS"] + p1_c1,
-            ["POSIX"] + posix2])
-
-        copy_list.append([
-            "POSIX -> UUID",
-            ["POSIX"] + posix1,
-            ["DAOS_UUID"] + p1_c2])
-
-        copy_list.append([
-            "POSIX -> UNS",
-            ["POSIX"] + posix1,
-            ["DAOS_UNS"] + p1_c2])
-
-        copy_list.append([
-            "POSIX -> POSIX",
-            ["POSIX"] + posix1,
-            ["POSIX"] + posix2])
-
-        # Run and verify each copy.
-        # Each src or dst is a list of params:
-        #   [param_type, path, pool, cont]
-        #   where the path is a directory.
+        copy_list = [
+            ["UUID -> UUID (same pool)", p1_c1_uuid, p1_c2_uuid],
+            ["LABEL -> LABEL (same pool)", p1_c1_label, p1_c2_label],
+            ["UUID -> UUID (different pool)", p1_c1_uuid, p2_c3_uuid],
+            ["UUID -> UNS (same pool)", p1_c1_uuid, p1_c2_uns],
+            ["UUID -> UNS (different pool)", p1_c1_uuid, p2_c3_uns],
+            ["UUID -> POSIX", p1_c1_uuid, posix2],
+            ["UNS -> UUID (same pool)", p1_c1_uns, p1_c2_uuid],
+            ["UNS -> UUID (different pool)", p1_c1_uns, p2_c3_uuid],
+            ["UNS -> UNS (same pool)", p1_c1_uns, p1_c2_uns],
+            ["UNS -> UNS (different pool)", p1_c1_uns, p2_c3_uns],
+            ["UNS -> POSIX", p1_c1_uns, posix2],
+            ["POSIX -> UUID", posix1, p1_c2_uuid],
+            ["POSIX -> UNS", posix1, p1_c2_uuid],
+            ["POSIX -> POSIX", posix1, posix2]
+        ]
+
+        # Run and verify each copy
         for (test_desc, src, dst) in copy_list:
             # dir -> dir variation
-            self.run_datamover(
-                test_desc + " (dir->dir)",
-                src[0], src[1], src[2], src[3],
-                dst[0], dst[1], dst[2], dst[3])
+            self.run_datamover(test_desc + " (dir->dir)", src_path=src, dst_path=dst)
 
             if self.tool == "DSYNC":
                 # The source directory is synced TO the destination.
-                dst_path = dst[1]
+                dst_path = dst
             else:
                 # The source directory is created IN the destination
                 # so append the directory name to the destination path.
-                dst_path = join(dst[1], basename(src[1]))
-            self.read_verify_location(dst[0], dst_path, dst[2], dst[3])
-
-            # The cases below use a UNS sub path, which is
-            # not supported by FS_COPY
-            if (self.tool == "FS_COPY" and src[0] == "DAOS_UNS" or dst[0] == "DAOS_UNS"):
-                continue
+                if 'daos:' in src:
+                    _, _, path = parse_path(src)
+                    path = path or '/'
+                else:
+                    path = src
+                dst_path = join(dst, basename(path))
+            self.log.info(f'src = {src}')
+            self.log.info(f'dst = {dst}')
```

Comment:
(pylint-logging-fstring-interpolation) Use lazy % formatting in logging functions